readthedocs/builds/models.py

```diff
@@ -247,13 +273,18 @@ def clean_build_path(self):
         try:
             path = self.get_build_path()
             if path is not None:
-                log.debug('Removing build path {0} for {1}'.format(path, self))
+                log.debug('Removing build path %s for %s', path, self)
                 rmtree(path)
         except OSError:
             log.exception('Build path cleanup failed')
 
     def get_github_url(
-            self, docroot, filename, source_suffix='.rst', action='view'):
+            self,
+            docroot,
+            filename,
+            source_suffix='.rst',
+            action='view',
```

Comment:
Here, the function is affected by `add-trailing-comma` plugin.

I want to check how to disable this case.

Reply:
This is not a problem of this plugin, but a combination of `add-trailing-comma` plus `yapf`.

When this is detected

```
 def myfunc(
        a, b, c, d, e, f, argument, larger, here, to, reach, eighty, lines, NOW
 ):
```

trailing comma adds a comma in the end, and yapf format it on an argument per line in the next pass.